pgcli/packages/expanded.py

```diff
@@ -1,10 +1,12 @@
+from .tabulate import _text_type
```

Comment:
Just an FYI, tabulate is a thrid party package (https://pypi.python.org/pypi/tabulate). I've copied the file physically since I added a new format that wasn't available. 

I've sent a PR to the original project and it's been merged. Waiting for their next release to start depending on it. 

So in the future we might not be able to do `from .tabulate import _text_type`. But this will do for now. :)